apps/adgerdir/screens/Article.tsx

```diff
@@ -53,46 +45,157 @@ interface ArticleProps {
   namespace: Query['getNamespace']
 }
 
-const simpleSpacing = [2, 2, 3] as ResponsiveSpace
-
 const Article: Screen<ArticleProps> = ({ article, pages, tags, namespace }) => {
-  const [contentOverviewOptions, setContentOverviewOptions] = useState([])
-  const { activeLocale } = useI18n()
-  // TODO: get language strings from namespace...
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const n = useNamespace(namespace)
-  const { makePath } = useRouteNames(activeLocale as Locale)
 
-  // const { fields: articleFields } = article
   const { items: pagesItems } = pages
   const { items: tagsItems } = tags
 
+  /* const statusNames = {
+    preparing: 'Í undirbúningi',
+    ongoing: 'Í framkvæmd',
+    completed: 'Lokið',
+  } */
+
+  if (!article) {
+    return (
+      <>
+        <Head>
+          <title>404 | Viðspyrna fyrir Ísland</title>
+        </Head>
+        <ArticleLayout>
+          <Stack space={3}>
+            <Breadcrumbs color="blue400">
+              <Link as="/" href="/">
+                <a>404</a>
+              </Link>
+            </Breadcrumbs>
+            <Box>
+              <Typography variant="h1" as="h1">
+                Síða fannst ekki!
+              </Typography>
+            </Box>
+          </Stack>
+        </ArticleLayout>
+      </>
+    )
+  }
+
+  const { estimatedCostIsk, finalCostIsk } = article
+
+  const estimatedCostFormatted =
+    estimatedCostIsk && formatNumberToKr(estimatedCostIsk)
+  const finalCostFormatted = finalCostIsk && formatNumberToKr(finalCostIsk)
+
+  const description = article.longDescription || article.description
+
   return (
     <>
       <Head>
-        <title>{article.title} | Ísland.is</title>
+        <title>{article.title} | Viðspyrna fyrir Ísland</title>
       </Head>
-      <ArticleLayout sidebar={<div>ok</div>}>
+      <ArticleLayout
+        sidebar={
+          <Box marginBottom={10}>
+            <Stack space={3}>
+              {article.link ? (
+                <Button icon="external" width="fluid" href={article.link}>
+                  {article.linkButtonText ?? n('seeMoreDetails')}
+                </Button>
+              ) : null}
+              {/* <Stack space={1}>
+                <Typography variant="tag" color="red600">
+                  Staða aðgerðar:
+                </Typography>
+                <Tag variant="red" label>
+                  <Box position="relative">
+                    <Inline space={1} alignY="center">
+                      <span>{statusNames[article.status]}</span>
+                      <span
+                        className={cn(
+                          cardStyles.status,
+                          cardStyles.statusType[article.status],
+                        )}
+                      ></span>
+                    </Inline>
+                  </Box>
+                </Tag>
+              </Stack> */}
```

Comment:
There is commented out code in quite a few places. Is it ok to remove?

Reply:
I will keep this in for now